.github/workflows/build.yml

```diff
@@ -96,8 +96,8 @@ jobs:
       - name: checkout Quarkus repository
         uses: actions/checkout@v2
         with:
-          repository: quarkusio/quarkus
-          ref: main
+          repository: mskacelik/quarkus
+          ref: srgql-tck-fix
```

Comment:
could you name it `srgql-2.9.1` or something? It's quite likely that we will need to add more changes to that branch...